backend/test/acceptance/__snapshots__/api.shoots.spec.js.snap

```diff
@@ -981,7 +1037,7 @@ users:
 }
 `;
 
-exports[`api shoots should return shoot seed info when no fallback is needed 1`] = `
+exports[`api shoots should return shoot seed info when need to fallback to old monitoring secret 1`] = `
```

Comment:
Why is this changed. Has the order changed? Can we revert it?

Reply:
It seems that the initial order of the snapshots https://github.com/gardener/dashboard/pull/1178/files#diff-7e96d3311ad28585b977e916709b718e22e93aaac7e42ad86b450697722355f9 was wrong. Only explanation I have the entries have been added manually.

Reply:
so I guess we go for the generated order, right?

Reply:
yes